diff --git a/node_modules/oceanic.js/dist/lib/structures/Guild.js b/node_modules/oceanic.js/dist/lib/structures/Guild.js
index fc7a6f5..11051d8 100644
--- a/node_modules/oceanic.js/dist/lib/structures/Guild.js
+++ b/node_modules/oceanic.js/dist/lib/structures/Guild.js
@@ -293,24 +293,7 @@ class Guild extends Base_1.default {
     }
     // true = `memberCount`
     updateMemberLimit(toAdd) {
-        const original = this.members.limit;
-        const num = toAdd === true ? this.memberCount : this.members.limit + toAdd;
-        const round = 10 ** (Math.floor(Math.log10(num)) - 1);
-        if (toAdd === true) {
-            const limit = Math.round(num / round) * round + round;
-            if (this.members.limit >= limit) {
-                return;
-            }
-            this.members.limit = limit;
-        }
-        else {
-            const limit = Math.round((this.members.size + toAdd) / round) * round + round;
-            if (this.members.limit >= limit) {
-                return;
-            }
-            this.members.limit = limit;
-        }
-        this.client.emit("debug", `The limit of the members collection of guild ${this.id} has been updated from ${original} to ${this.members.limit} to accommodate at least ${toAdd === true ? this.memberCount : this.members.size + toAdd} members.`);
+    return; // no-op
     }
     update(data) {
         if (data.afk_channel_id !== undefined) {
